website/content/docs/job-specification/service.mdx

```diff
@@ -141,9 +141,12 @@ Connect][connect] integration.
   [documentation](https://www.consul.io/docs/internals/anti-entropy#enable-tag-override)
   for more information. Only available where `provider = "consul"`.
 
+- `address` `(string: <optional>)` - Specifies a custom address to advertise in
+  Consul or Nomad service registration. Can be an IP address or domain name. If
+  set, `address_mode` must be in `auto` mode.
```

Comment:
Let's note that interpolation is supported or perhaps even suggest using it. Perhaps:

> - `address` `(string: <optional>)` - Manually specifies a custom address to advertise. `address_mode` must be left on its default `auto` mode. This is most useful in combination with interpolation. For example to advertise the public IP address of an AWS EC2 node set this to `${unique.platform.aws.public-ipv4}`